mmv1/third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -14,6 +15,22 @@ import (
 	"google.golang.org/api/bigquery/v2"
 )
 
+func sortArrayByName(array []interface{}) {
```

Comment:
Can you namespace these under bigquery table? They're generally named when we probably don't want them to get reused.

Reply:
done